crypto/crypto_aesctr_aesni.c

```diff
@@ -0,0 +1,230 @@
+#include "cpusupport.h"
+#ifdef CPUSUPPORT_X86_AESNI
+
+#include <assert.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <emmintrin.h>
+
+#include "crypto_aes_aesni.h"
+#include "insecure_memzero.h"
+#include "sysendian.h"
+
+#include "crypto_aesctr_aesni.h"
+
+struct crypto_aesctr {
+	const struct crypto_aes_key * key;
+	uint64_t bytectr;
+	__m128i bufsse;
```

Comment:
This is illegal; that's the whole reason we have the align-ptr branch.  So I need to fix the bugs in that, before moving forward with this one.